src/main/java/Task.java

```diff
@@ -0,0 +1,40 @@
+package main.java;
+
+public class Task {
+    private String name;
+    private boolean isDone;
+
+    public Task() {
+        name = "";
+        isDone = false;
+    }
+
+    public Task(String n, boolean d) {
+        name = n;
+        isDone = d;
+    }
```

Comment:
public Task(String name, boolean isDone) {
        this.name = name;
        this.isDone = isDone;
    }